packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -101,8 +100,7 @@ const Onboarding = () => {
         })}
         onClick={handleClick}
       >
-        {!isOpen && <FontAwesomeIcon icon={faQuestion} />}
-        {isOpen && <FontAwesomeIcon icon={faTimes} />}
+        {isOpen ? <Cross /> : <Question />}
```

Comment:
I wonder if we should wrap the icons in the [`Icon` component](https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-technical-components-icon--base) to apply the same style to Cross and Question icons (Questions seem quite small compared to Cross):
<img width="219" alt="image" src="https://user-images.githubusercontent.com/71838159/208700491-1257872c-629f-4cb4-aa97-64c6c3de3884.png">
<img width="193" alt="image" src="https://user-images.githubusercontent.com/71838159/208700535-ec983942-140e-49c1-b237-2433d17146a6.png">

wdyt?